packages/textInput/components/TextInput.tsx

```diff
@@ -43,7 +43,7 @@ export interface TextInputProps extends React.HTMLProps<HTMLInputElement> {
   /**
    * Sets the contents of the input label. This can be a `string` or any `ReactNode`.
    */
-  inputLabel: React.ReactNode;
+  inputLabel?: React.ReactNode;
```

Comment:
Why do we make this optional?

Reply:
because we often times don't set it. specially when `showInputLabel` is set to false. maybe we can change the API some day, so that becomes more clear.

Reply:
Okay, you convinced me. :D
@mperrotti , what do you think?

Reply:
the more prevalent problem here is that we don't test the types of our tests. without this change tests threw type errors at me.